### ISSUE TITLE
[Feature request] Files management

### ISSUE THREAD

**Issue by uKL** [amitshekhariitbhu/Android-Debug-Database#58]
Hey,

great tool! I treat it as a central place to get data from the app during my development. Although it would be nice to have files management as well. Files stored in the application's *files* directory could be:

- viewable
- deletable
- uploadable

What do you think about it? Do you have any plans? Is this kind of contribution possibly welcome?

**Comment by amitshekhariitbhu** [comment]
@uKL Yes you are correct, actually we have planned for this. Let's build it together.

Thank you
Amit Shekhar

**Comment by Keirnan** [comment]
How did I even get subscribed to this one... Lol.
